core/src/main/scala/kafka/raft/RaftManager.scala

```diff
@@ -16,15 +16,10 @@
  */
 package kafka.raft
 
-import java.io.File
-import java.nio.file.Files
-import java.util.Random
-import java.util.concurrent.CompletableFuture
-
 import kafka.log.{Log, LogConfig, LogManager}
 import kafka.raft.KafkaRaftManager.RaftIoThread
 import kafka.server.KafkaServer.ControllerRole
-import kafka.server.{BrokerTopicStats, KafkaBroker, KafkaConfig, LogDirFailureChannel, MetaProperties}
+import kafka.server._
```

Comment:
Do we normally allow wildcard imports? I think we might not

Reply:
Not sure. Checkstyle didn't complain. So, I assumed it was allowed. Also, it was IntelliJ's "Optimize Imports" that did this. I can go bump up the number of imports before replacing it with a wildcard if it's preferred to not have wildcards.